stf_functional_tests.yml

```diff
@@ -17,73 +16,26 @@
   hosts: overcloud_nodes
   ignore_errors: true
   become: true
+  roles:
```

Comment:
I feel like use of `roles` is out of favour with a preference to doing an `import_role` or `include_role` as a task.

In which case, I think you could do a lookup of the files and iterate over the list with `loop` and `include_role: "{{ item }}"`. 

Not a big deal, more a style thing.

Reply:
The role is imported here to do set-up, so that the defaults are available to the tasks being run.

There's not anything to loop through really.

Reply:
Well I was just saying you can loop through the roles by importing them via `import_role` vs the `roles` parameter which seems to be falling out of favour.

Ultimately this is a style decision though, and I have no strong preference here. If it works as-is, good enough :)